lib/src/cli/cli.dart

```diff
@@ -0,0 +1,43 @@
+import 'package:path/path.dart' as p;
+import 'package:universal_io/io.dart';
+
+part 'flutter_cli.dart';
+
+/// Abstraction for running commands via command-line.
+class _Cmd {
+  /// Runs the specified [cmd] with the provided [args].
+  static Future<ProcessResult> run(
```

Comment:
Are you fully convinced to run commands via static methods? Maybe having a CommandClient or something similar would be more inline with our general approach to modularization?

Reply:
Yeah that's a larger refactor we can prioritize. My primary goal for this PR was to make it so we could easily introduce a `dart_cli.dart` to support `dart fix`. The existing `_Cmd` API is not affected by these changes but I think we can create a separate PR to address that. wdyt?

Reply:
sounds good, just wanted to chime in but I'm ok with the current approach